v5/types.go

```diff
@@ -148,6 +148,7 @@ type Customer struct {
 	BrowserID                    string            `json:"browserId,omitempty"`
 	MgCustomerID                 string            `json:"mgCustomerId,omitempty"`
 	PhotoURL                     string            `json:"photoUrl,omitempty"`
+	Tags                         []string          `json:"tags,omitempty"`
```

Comment:
Actual tags field in the response looks like this:
```json
[
      {
        "name": "first",
        "color": "#3e89b6",
        "attached": false
      },
      {
        "name": "second",
        "color": "#ffa654",
        "attached": false
      }
]
```

Latest documentation doesn't reflect this for now. In order to implement tags field in the customer type you should create another struct, which may be called `Tag`, and replace string type for the `Tags` slice with implemented `Tag` struct.

Also, don't forget to leave godoc comment for a new struct - format should be same to all other godoc strings in this file. There is no need to document this in detail - we have API documentation for that.